vital/klv/convert_0104_metadata.cxx

```diff
@@ -145,7 +155,7 @@ ::convert_0104_metadata( klv_uds_vector_t const& uds, metadata& md )
   // lat-lon points and image corner points. All geodetic points are assumed to
   // be WGS84 lat-lon.
```

Comment:
We should specify the units we are using in these description comments (degrees and meters)